src/Backups/BackupCoordinationRemote.cpp

```diff
@@ -224,37 +234,49 @@ zkutil::ZooKeeperPtr BackupCoordinationRemote::getZooKeeperNoLock() const
 
 void BackupCoordinationRemote::createRootNodes()
 {
-    auto zk = getZooKeeper();
-    zk->createAncestors(zookeeper_path);
-    zk->createIfNotExists(zookeeper_path, "");
-    zk->createIfNotExists(zookeeper_path + "/repl_part_names", "");
-    zk->createIfNotExists(zookeeper_path + "/repl_mutations", "");
-    zk->createIfNotExists(zookeeper_path + "/repl_data_paths", "");
-    zk->createIfNotExists(zookeeper_path + "/repl_access", "");
-    zk->createIfNotExists(zookeeper_path + "/file_names", "");
-    zk->createIfNotExists(zookeeper_path + "/file_infos", "");
-    zk->createIfNotExists(zookeeper_path + "/archive_suffixes", "");
+    auto zookeeper_retries_info = global_zookeeper_retries_info;
+    ZooKeeperRetriesControl retries_ctl("createRootNodes", zookeeper_retries_info);
+    retries_ctl.retryLoop(
```

Comment:
I did it on purpose. Having small amount of retries with independent backoffs is better than having a huge amount of retries for the whole backup (and it should be huge since backup can take long time to finish).